templates/hotgraphic.hbs

```diff
@@ -66,7 +66,11 @@
         {{/if}}
         {{#each _items}}
           <button class="base hotgraphic-graphic-pin component-item-text-color item-{{@index}} {{#if _isVisited}}visited{{/if}}" data-id="item-{{@index}}" style="top:{{{_top}}}%; left:{{{_left}}}%;" role="button" aria-label="Item {{numbers @index}}. {{title}}.{{#if _isVisited}} {{../../_globals._accessibility._ariaLabels.visited}}.{{/if}}">
+            {{#if ../_useNumbersAsPins}}
+            <div class="hotgraphic-graphic-number-icon component-item-color item-{{@index}}">{{inc @index}}</div>
```

Comment:
```inc @index``` just means add one to the ```@index``` of the item and output that text. this is as the items array start at index 0 and that doesn't make much sense to a user, they need the text "1,2,3,4,5, etc"

the class "item-0" is for styling.

i feel as though i've missed the point...